car/car.ino

```diff
@@ -129,3 +122,85 @@ void handleInput() { //handle serial input (String!!)
     }  
   }
 }
+
+void driveAroundObstical(){
+  int leftDistance = left.readRangeContinuousMillimeters();
+  int rightDistance = right.getDistance();
+  
+  if (rightDistance <= RIGHT_DIST && rightDistance > 0){     //Turn left if obstical on the right 
```

Comment:
There is a LOT of duplicated code in this if-else if statements.

Reply:
Yes. We wanted to make it work, and then make the code neat and nice :)